app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,268 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+ * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any styles
+ * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
+ * file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+ @import "font-awesome";
+ @import "bootstrap-sprockets";
+ @import "bootstrap";
+
+ body {
+   font-family: 'Quicksand', sans-serif;
+ }
+
+.thumbnail {
+  width: 300px;
+  height: 350px;
+  overflow: hidden;
+}
+
+.thumbnail-container {
+  width: 300px;
+  height: 300px;
+  overflow: hidden;
+  margin:auto;
+}
+
+.thumbnail-image {
+  width: 100%;
+  height: auto;
+}
+
+.cart-image {
+  width: 100%;
+  height: auto;
+}
+
+.cart-image-container {
+  width: 150px;
+  height: 150px;
+  overflow: hidden;
+}
+
+.nav-header {
+  font-family: 'Raleway', sans-serif;
+  font-size: 60px;
+  color: #000000;
+}
+
+.nav-header:hover, .nav-header:active, .cat-link:hover, .nav-link:hover {
+  text-decoration: none;
+  color: #006666;
+}
+
+.product-image-container {
+  width: 300px;
+  height: 300px;
+  overflow: hidden;
+  margin: auto;
+}
+
+.product-image {
+  width: 100%;
+  height: auto;
+}
+
+.cat-link {
+  color: #000000;
+  font-size: 12px;
+  font-family: 'Quicksand', sans-serif;
+}
+
+.cat-link-container {
+  display: inline-block;
+  margin-left: 20px;
+}
+
+// .cat-container {
```

Comment:
It's good practice to delete any commented out code you're not using. I get that y'all don't have _a ton_ of time, but just as an fyi.